### ISSUE TITLE
Player dies on disconnect

### ISSUE THREAD

**Issue by nerdyjohnny** [nerdyjohnny/ovicommunity#6]
Player dies when disconnecting. Ticket 23

**Comment by nerdyjohnny** [comment]
Unable to reproduce the issue. KickQuitLister will monitor into logs incase it happens again.

**Comment by NomiSky** [comment]
Issue has happened again but only once.

**Comment by nerdyjohnny** [comment]
```
[04:21:16] [Server thread/INFO]: NomiSky lost connection: Disconnected
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [LOWEST]: NomiSky's HP = 20.726303100585938
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [LOW]: NomiSky's HP = 20.726303100585938
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [NORMAL]: NomiSky's HP = 20.726303100585938
[04:21:16] [Server thread/INFO]: NomiSky died
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [HIGH]: NomiSky's HP = 0.0
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [HIGHEST]: NomiSky's HP = 0.0
[04:21:16] [Server thread/INFO]: PlayerQuitEvent [MONITOR]: NomiSky's HP = 0.0
[04:21:16] [Server thread/INFO]: PlayerQuitEvent Listeners:
[04:21:16] [Server thread/INFO]:  - NORMAL: WorldEdit
[04:21:16] [Server thread/INFO]:  - NORMAL: PlaceholderAPI
[04:21:16] [Server thread/INFO]:  - NORMAL: AugmentedHardcore
[04:21:16] [Server thread/INFO]:  - NORMAL: LibsDisguises
[04:21:16] [Server thread/INFO]:  - NORMAL: Scoreboard-revision
[04:21:16] [Server thread/INFO]:  - NORMAL: AdvancedBan
[04:21:16] [Server thread/INFO]:  - NORMAL: WorldGuard
[04:21:16] [Server thread/INFO]:  - NORMAL: DeluxeAnnounce
[04:21:16] [Server thread/INFO]:  - NORMAL: DeluxeAnnounce
[04:21:16] [Server thread/INFO]:  - NORMAL: DeluxeAnnounce
[04:21:16] [Server thread/INFO]:  - NORMAL: DeluxeAnnounce
[04:21:16] [Server thread/INFO]:  - HIGHEST: Essentials
[04:21:16] [Server thread/INFO]:  - HIGHEST: Spartan
[04:21:16] [Server thread/INFO]:  - MONITOR: LuckPerms
[04:21:16] [Server thread/INFO]:  - MONITOR: LuckPerms
[04:21:16] [Server thread/INFO]:  - MONITOR: ProtocolLib
[04:21:16] [Server thread/INFO]:  - MONITOR: PlaceholderAPI
[04:21:16] [Server thread/INFO]:  - MONITOR: EssentialsDiscord
[04:21:16] [Server thread/INFO]: NomiSky left the game
```

**Comment by nerdyjohnny** [comment]
Issue is only apparent on disconnects. Still unable to diagnose issue. Issue is isolated to HCMC so it might be due to conflicting player data when the server was established from the RP server's data

**Comment by nerdyjohnny** [comment]
Issue will be put On Hold until the new update is published unless we diagnose the issue. Community is welcomed to leave ideas on what this issue might be.

**Comment by nerdyjohnny** [comment]
It appears the way to address the issue is to reset everyone's player data as this has been a progressive issue for multiple players. With the release of 1.18, the world is going to need regenerated due to the way 1.18 handles world generation. I think a complete wipe will be the solution to everything.

**Comment by nerdyjohnny** [comment]
Complete wipe preformed
